### ISSUE TITLE
flask.ext.cache throws deprecation warnings 

### ISSUE THREAD

**Issue by deidyomega** [thadeusb/flask-cache#142]
When using flask_cache:

ExtDeprecationWarning: Importing flask.ext.cache is deprecated, use flask_cache instead.

Even if user code is correct (using flask_cache instead of flask.ext.cache) the warning is still thrown.  I suspect this is because internally an import to flask.ext.cache is called, thus spawning the warning.

**Comment by woozyking** [comment]
This starts from Flask `0.11`

**Comment by dappiu** [comment]
I didn't saw the issue before, but just created a pull request for this #144
It's a while that this import format is reported as deprecate (since the long standing 0.10 for sure), I think is relatively safe to change to the new format

**Comment by ar-anvd** [comment]
+1 for this issue.

**Comment by deidyomega** [comment]
I created a pull request for this, however this pull is the best one I've seen out of the group to resolve this issue:

https://github.com/thadeusb/flask-cache/pull/139

**Comment by bart3005** [comment]
Just wondering is this likely to be fix?

**Comment by deidyomega** [comment]
Off what @bart3005  said, is anyone actively maintaining this package?  Last commit was Sep 21, 2014.

**Comment by bart3005** [comment]
@deidyomega exactly what I was alluding to.

**Comment by xcash** [comment]
yeah looking at the author's profile he's not using github for a year... is anyone planning on contacting him for a takeover of the package?

**Comment by rudaporto** [comment]
+1 to make a release and upload package to pypi :-)

**Comment by bart3005** [comment]
I think this really needs taking over. I wonder if the owner would transfer?

**Comment by RSully** [comment]
I reached out over email with a link to this thread.

**Comment by sh4nks** [comment]
If anyone is interested, I forked this extension, merged a bunch of PRs and released it under a slightly different name ([Flask-Caching](https://github.com/sh4nks/flask-caching)).

All the merged PRs can be found in the changelog:
https://pythonhosted.org/Flask-Caching/#version-1-0-0-2016-07-05
